src/app.css

```diff
@@ -2,6 +2,51 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+    --theme-font-color-base: #383c40;
+    --theme-font-color-dark: #f7f9fa;
+    --theme-rounded-base: 8px;
+    --theme-rounded-container: 16px;
+    --theme-border-base: 1px;
+    --on-primary: #ffffff;
+    --on-secondary: #ffffff;
+    --on-tertiary: #ffffff;
+    --on-success: #000000;
+    --on-warning: #000000;
+    --on-error: #000000;
+    --on-surface: #ffffff;
+    --color-primary-50: #fdf1e9;
```

Comment:
Tailwind already generates all the utilities based on the colors that you added from line 18 until line 47. They're defined in the `tailwind.config.js` file. If you're not using the css variables somewhere else, I'd remove them from here